docs/cpp/platforms/macros.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Pre-Compiled Macros
+layout: docs
+sidenav: side-nav-cpp.html
+type: markdown
+---
+
+## Platform Check Macros
+
+C and C++ compilers automatically define some macros that can be used to:
+
+ * Identify platforms (architecture, endianness, OS, compiler,
+   C++ standard, library)
+ * Detect compiler configurations (debug mode, exception handling, etc.)
+
+Collectively, these macros are known as *pre-defined compiler macros*,
+and are used in Abseil to conditionally compile code for
+various purposes like portability, performance, functionality, etc. This
+document lists the compiler macros Abseil uses to write preprocessor
+conditionals and build configuration macros.
+
+This document is designed solely to document Abseil's usage of these
+macros, and assist developers who need to add architecture-specific
+code to Abseil.
+
+Note: [Feature Check Macros](feature_checks), which developers can use to
```

Comment:
Does this link work now? If it doesn't, is it OK?

Reply:
It works for me. Pls. check again on staging.

Reply:
it works for me.